src/components/Cards/card.tsx

```diff
@@ -45,12 +45,21 @@ export default function Card({ props }: { props: CardProps }) {
       role="presentation"
       className="group col-span-1 max-w-xs flex-col overflow-hidden rounded-lg bg-grey-100 shadow-md transition hover:-translate-y-0.5 hover:cursor-pointer hover:bg-grey-50"
     >
-      <Modal setShowPopup={setShowModal} showModal={showModal}>
+      <Modal
+        title={
+          <>
+            Repair ID
```

Comment:
It's ok to have "Repair ID" and the id on the same line because it'll be changing to an autoincrementing integer